app/views/shops/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>Input information for a new shop</h1>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for [@company, @shop], url:company_shops_path(@company), method: :post  do |shop_form | %>
+      <%= render 'shared/error_messages', object: shop_form.object %>
+      <div>
+        <%= shop_form.label :name %>
+        <%= shop_form.text_field :name %>
+
+        <%= shop_form.label :email %>
+        <%= shop_form.text_field :email %>
+      </div>
+      </br>
+      <div>
+       <%= shop_form.fields_for @address do |address_form| %>
+         <%= address_form.label :short_address %>
+         <%= address_form.text_field :short_address %>
+
+         <%= address_form.label :full_address %>
+         <%= address_form.text_field :full_address %>
+
+         <%= address_form.label :city %>
+         <%= address_form.text_field :city %>
+
+         <%= address_form.label :zipcode %>
+         <%= address_form.text_field :zipcode %>
+
+         <%= address_form.label :country %>
+         <%= address_form.text_field :country %>
+      <% end %>
```

Comment:
indentation. Use RUBOCOP to verify code standards in the whole project before every push